app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,12 @@
+.members-list {
+  max-height: 40em;
+  overflow-y: auto;
+}
+
+.add-member .field_with_errors {
+  padding: 0;
+  display: inline;
+  background-color: transparent;
+
+  @extend .has-error;
```

Comment:
We should keep the mixins on the top of the ruleset, what do you think? It makes it easier to read.